src/account/accountProfileUtils.js

```diff
@@ -56,12 +56,16 @@ export const useCompleteProfile = () => {
   const location = useLocation();
   const navigate = useNavigate();
   const { data: user } = useSelector(state => state.account.currentUser);
-  const [isFirstLogin, setIsFirstLogin] = useState();
+  const [isFirstLogin, setIsFirstLogin] = useState(false);
 
   useEffect(() => {
-    getIsFirstLogin().then(isFirstLogin => {
-      setIsFirstLogin(isFirstLogin);
-    });
+    getIsFirstLogin()
+      .then(isFirstLogin => {
+        setIsFirstLogin(isFirstLogin);
+      })
+      .catch(() => {
+        setIsFirstLogin(false);
+      });
```

Comment:
question: i'm not understanding what's going on here: trying to follow the code, it seems that since the `useEffect` has no dependencies it will only run once, and since `isFirstLogin` is now default `false`, `setIsFirstLogin(false)` should have no effect? i'm also not sure how this is related to the optional auth fix